### PR TITLE
2.2.0 release notes (with 1 pending change)

### PR DESCRIPTION
**Pull request by janl**
Let’s start writing up the 2.2.0 release notes.

### CODE REVIEW

**Comment by wohali** [comment]
@janl the noise above is a rebase against master... commit 2ab4aa9 is the one you want to look at for just the notes so far. I'm about halfway done, hope to get more done late tonight.

File: src/whatsnew/2.2.rst [hunk 1]
```diff
@@ -27,30 +27,173 @@ Upgrade Notes
...
+
+* The maximum http request limit, which had been lowered in 2.1.0, has been re-raised
+  to a 4GB limit for now. (:ghissue:`1446`). Ongoing discussion about the path forward
+  for future releases is available in :ghissue:`1200` and :ghissue:`1253`.
+
+* :ghissue:`1153`: The CouchDB replicator can now make use of the ``/_session`` endpoint
+  rather than relying entirely on HTTP basic authentication headers. This can greatly
+  improve replication performance. We encourage you to upgrade any nodes or clusters that
+  regularly act as replication clients to use this new feature, which is enabled by
+  default (:ghissue:`1462`).
```
Comment (wohali) [inline src/whatsnew/2.2.rst]: N.B. This PR has not yet merged - if we decide not to push it in 2.2.0, I will update this to reflect that reality.

File: src/whatsnew/2.2.rst [hunk 2]
```diff
@@ -27,30 +27,283 @@ Upgrade Notes
 
 .. rst-class:: open
 
-* Item 1
+* The minimum supported version of Erlang is now R17, not 16B03. Support for Erlang 21
+  is still ongoing and will be provided in a future release.
 
-* Item 2
+* TODO WORDING: Recommend increasing PBKDF2 / bcrypt work factor and auth_plugins to
+  couch_replicator_auth_session,couch_replicator_auth_noop. Do not do this, if an older
```
Comment (nickva) [inline src/whatsnew/2.2.rst]: There is a PR to make `couch_replicator_auth_session,couch_replicator_auth_noop` the default. 

https://github.com/apache/couchdb/pull/1462

If that goes into 2.2 we can omit the part about this config.

 Also we'd want to remove bcrypt reference as it was backed out due to Windows build failures.

Comment (wohali) [reply src/whatsnew/2.2.rst]: The PR has landed in 2.2 and I have rewritten the section.

**Comment by wohali** [comment]
Something is wrong with this branch...I'm going to close this and try again. Sorry :(

**Pull request closed without merging**